docs/en/tutorials/getting-started.rst

```diff
@@ -372,22 +367,19 @@ Call this script from the command-line to see how new products are created:
     $ php create_product.php ORM
     $ php create_product.php DBAL
 
-What is happening here? Using the ``Product`` is pretty standard OOP.
+What is happening here? Using the ``Product`` class is pretty standard OOP.
 The interesting bits are the use of the ``EntityManager`` service. To
-notify the EntityManager that a new entity should be inserted into the database
-you have to call ``persist()``. To initiate a transaction to actually perform
-the insertion, You have to explicitly call ``flush()`` on the ``EntityManager``.
+notify the EntityManager that a new entity should be inserted into the database,
+you have to call ``persist()``. To initiate a transaction to actually *perform*
+the insertion, you have to explicitly call ``flush()`` on the ``EntityManager``.
 
-This distinction between persist and flush is allows to aggregate all writes
-(INSERT, UPDATE, DELETE) into one single transaction, which is executed when
-flush is called. Using this approach the write-performance is significantly
-better than in a scenario where updates are done for each entity in isolation.
+This distinction between persist and flush is what allows the aggregation of
+all database writes (INSERT, UPDATE, DELETE) into one single transaction, which
+is executed when ``flush()`` is called. Using this approach, the write-performance
+is significantly better than in a scenario in which writes are performed on
+each entity in isolation.
 
```

Comment:
Introducing the UnitOfWork concept here seems premature. It's been moved down to the next section.